.github/workflows/rss-runner.yaml

```
@@ -23,20 +23,72 @@
+        run: |
+          source .virtualenv/bin/activate
+          python -m data_collections.mainRSSRunner
+
+      - name: Update runningCSV.csv with Other Events
+        env:
+          TASK_TYPE: "OTHER"
+          WORKSHOP_RSS: ${{ secrets.OTHER_RSS }}
+        run: |
+          source .virtualenv/bin/activate
+          python -m data_collections.mainRSSRunner
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)